SingularityUI/app/templates/requestTypeFilter.hbs

```diff
@@ -1,36 +1,32 @@
 <ul class="nav nav-pills nav-pills-multi-select">
     <li {{#unlessInSubFilter 'SERVICE' requestsSubFilter}}class="active"{{/unlessInSubFilter}}>
-        <a data-filter="SERVICE">
+        <a data-filter="SERVICE" class='has-tooltip' title="Protip: press cmd to select multiple" data-toggle='tooltip'>
             <span class="glyphicon glyphicon-ok"></span>
             Service
         </a>
     </li>
     <li {{#unlessInSubFilter 'WORKER' requestsSubFilter}}class="active"{{/unlessInSubFilter}}>
-        <a data-filter="WORKER">
+        <a data-filter="WORKER" class='has-tooltip' title="Protip: press cmd to select multiple" data-toggle='tooltip'>
             <span class="glyphicon glyphicon-ok"></span>
             Worker
         </a>
     </li>
     <li {{#unlessInSubFilter 'SCHEDULED' requestsSubFilter}}class="active"{{/unlessInSubFilter}}>
-        <a data-filter="SCHEDULED">
+        <a data-filter="SCHEDULED" class='has-tooltip' title="Protip: press cmd to select multiple" data-toggle='tooltip'>
             <span class="glyphicon glyphicon-ok"></span>
             Scheduled
         </a>
     </li>
     <li {{#unlessInSubFilter 'ON_DEMAND' requestsSubFilter}}class="active"{{/unlessInSubFilter}}>
-        <a data-filter="ON_DEMAND">
+        <a data-filter="ON_DEMAND" class='has-tooltip' title="Protip: press cmd to select multiple" data-toggle='tooltip'>
             <span class="glyphicon glyphicon-ok"></span>
             On-demand
         </a>
     </li>
     <li {{#unlessInSubFilter 'RUN_ONCE' requestsSubFilter}}class="active"{{/unlessInSubFilter}}>
-        <a data-filter="RUN_ONCE">
+        <a data-filter="RUN_ONCE" class='has-tooltip' title="Protip: press cmd to select multiple" data-toggle='tooltip'>
```

Comment:
Wondering if this is necessary, is there any reason they can't just be a toggle? Instead of having to hold down cmd, just clicking toggles that filter on/off, and all are selected by default to start with